db/migrate/20150709125746_create_albums.rb

```diff
@@ -1,12 +1,12 @@
-# class CreateAlbums < ActiveRecord::Migration
-#   def change
-#     create_table :albums do |t|
-#       t.string :title
-#       t.string :artist
-#       t.string :genre
-#       t.integer :length
-#       t.timestamps null: false
-#     end
-#   end
-#
-# end
+class CreateAlbums < ActiveRecord::Migration
+  def change
+    create_table :albums do |t|
+      t.string :title
+      t.string :artist
+      t.string :genre
+      t.integer :length
```

Comment:
if these all have presence validations on them, they should also probably be `null: false`, so like
```ruby
t.string :title, null: false
```